test/seed.rb

```diff
@@ -24,14 +24,14 @@ def self.add_wikirate_data
     Card.create! :name=>'Death Star+Force', :type=>'analysis', :subcards=> {
       '+article'  => { :content=>"I'm your father!" }
     }
-    
-    sourcepage = Card.create! :type_id=>Card::SourceID, :subcards=>{ 
+
+    sourcepage = Card.create! :type_id=>Card::SourceID, :subcards=>{
       '+Link' => {:content=> 'http://www.wikiwand.com/en/Star_Wars'},
       '+company' => { :content=>"[[Death Star]]",         :type_id=>Card::PointerID },
       '+topic'   => { :content=>"[[Force]]",              :type_id=>Card::PointerID }
     }
     Card.create! :name=>'Death Star uses dark side of the Force', :type_id=>Card::ClaimID, :subcards=> {
-      '+source'  => { :content=>"[[#{sourcepage.name}]]", :type_id=>Card::PointerID }, 
+      '+source'  => { :content=>"[[#{sourcepage.name}]]", :type_id=>Card::PointerID },
```

Comment:
Line is too long. [86/80]<br>Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.